### DISCUSSION TITLE
Hindi text not working properly

### DISCUSSION THREAD

**Discussion by AashishChakravarty** [ImageMagick/ImageMagick#6794] [Help]
### ImageMagick version

 7.1.1-16

### Operating system, version and so on

MacOS 14.0

### Description

I am using ImageMagick tool for creating some images with hindi Text but it's not working properly.

Fonts: https://fonts.google.com/noto/specimen/Noto+Sans+Devanagari

I have tried with below mentioned command.
```
convert sample.jpg -fill red -pointsize 40 -font NotoSansDevanagari.ttf -annotate +100+100 "आपका हिन्दी टेक्स्ट" output_image.jpg
```

## Wrong Output
![image](https://github.com/ImageMagick/ImageMagick/assets/29206561/a85cde9e-69b3-4411-bb6d-26f86d2e7043)

## Expected Output
![image](https://github.com/ImageMagick/ImageMagick/assets/29206561/57ba909f-5762-4255-b850-ef4fc077f0e0)

What I missed in the command to get the expected output?

**Comment by fmw42** [comment]
You cannot just type your text. You need to use UTF-8 compatible text editor. See https://imagemagick.org/Usage/text/#unicode
